support/comments.py

```diff
@@ -0,0 +1,23 @@
+import requests
```

Comment:
files from different layers of TAF are in the support directory, should be separated